source/common/grpc/google_async_client_cache.h

```diff
@@ -29,9 +29,8 @@ class AsyncClientCache : public std::enable_shared_from_this<AsyncClientCache> {
   struct ThreadLocalCache : public ThreadLocal::ThreadLocalObject {
     ThreadLocalCache(AsyncClientManager& async_client_manager, Stats::Scope& scope,
                      const ::envoy::config::core::v3::GrpcService& grpc_proto_config) {
-      const AsyncClientFactoryPtr factory =
-          async_client_manager.factoryForGrpcService(grpc_proto_config, scope, true);
-      async_client_ = factory->create();
+      async_client_ =
```

Comment:
Are `AsyncClientCache` and `AsyncClientCacheSingleton` even needed anymore?

Reply:
No, they are just legacy code, I can clean them up.